spec/services/idv/usps_mail_spec.rb

```diff
@@ -44,4 +44,31 @@
       end
     end
   end
+
+  def event_create(hash)
```

Comment:
This method looks like it is doing some work that could be moved to a factory. Thoughts on moving it to one?